crates/analytics/src/clickhouse.rs

```diff
@@ -360,7 +373,7 @@ impl ToSql<ClickhouseClient> for AnalyticsCollection {
             Self::Payment => Ok("payment_attempts".to_string()),
             Self::Refund => Ok("refunds".to_string()),
             Self::SdkEvents => Ok("sdk_events_audit".to_string()),
-            Self::ApiEvents => Ok("api_events_audit".to_string()),
+            Self::ApiEvents | Self::AuthEvents => Ok("api_events_audit".to_string()),
```

Comment:
`api_events_audit` is a ID based table , cannot use for time based filtering

If want can use `api_events`